### ISSUE TITLE
The Visitor Map does not show unique visitors at city level

### ISSUE THREAD

**Issue by eramirezprotec** [matomo-org/matomo#12860]
Is it the normal behavior of the Visitor Map to not show unique visitors when city level is activated?
I see this happening in the demo, but I'm not sure if it is intended or an error.

Thank you very much.

**Comment by Findus23** [comment]
So you mean adding a `nb_uniq_visitors: 0.002`-line here:
https://github.com/matomo-org/matomo/blob/6aaea44441b20b94707498c34b2213df704af2b1/plugins/UserCountryMap/javascripts/visitor-map.js#L903-L909

**Comment by eramirezprotec** [comment]
I'm sorry, I edited really bad that comment... the line is the one specifying the value for unique visitors metric.

**Comment by eramirezprotec** [comment]
Exactly, @Findus23

**Comment by eramirezprotec** [comment]
Hi!

In its moment, unique visitors' points were not visible on the Visitor Map when it showed the cities. 

After we corrected that error (the reason for this ticket to be created), we didn't check that everything was working perfectly. Now, one and a half years later, we noticed that the "big points" containing the largest numbers, were not visible on the map. 

By making the following change to the file plugins\UserCountryMap\javascripts\visitor-map.js :

```
groups[g_id] = {
  nb_visits: 0,
  nb_uniq_visitors: 0, // this line
  nb_actions: 0,
  sum_visit_length: 0,
  bounce_count: 0
};
```

...the big points are finally being shown.

https://github.com/matomo-org/matomo/blob/2c5c72e2da4c015efa78069c9c1b407fc55a6483/plugins/UserCountryMap/javascripts/visitor-map.js#L676-L681
